nixos/modules/programs/shadow.nix

```diff
@@ -6,17 +6,18 @@ with lib;
 
 let
 
+  # Note that these id ranges are duplicated in update-users-groups.pl
```

Comment:
Is there a place where we document when `login.defs` gets used, and when `update-users-groups.pl`?

If not, this could be an opportunity to do it. Maybe even just expanding the inline comment a bit could be enough, explaining why the duplication is in place.

Reply:
Yeah that's a good idea, will do